src/averageAges.js

```diff
@@ -14,12 +14,24 @@
  *
  * @return {number}
  */
+
+const calculateAverageAge = (age, people) => {
+  return age.reduce((a, b) => a + b) / people.length;
+};
```

Comment:
1. Why do you need pass people array if `age.length === people.lengt`h already?
2. Again age is a bad name
```suggestion
const calculateAverageAge = (ages) => {
  return ages.reduce((a, b) => a + b) / ages.length;
};
```